src/momento/auth/credential_provider.py

```diff
@@ -44,7 +44,7 @@ def from_environment_variable(
 
     @staticmethod
     def from_string(
-        auth_token: str,
+        auth_token: Union[bytes, str],
```

Comment:
why do we need to support bytes here?

Reply:
This was also motivated by IDE warnings and was mostly intended to be defensive. After doing some testing though I'll go ahead and remove it as it's more complicated than needed.